features/upgrade/api_auth/upgrade.feature

```diff
@@ -351,29 +351,3 @@ Feature: apiserver and auth related upgrade check
     # So we just add a simple/useless step here to get rid of the errors in the log.
     Given the expression should be true> "True" == "True"
 
-  # @author xxia@redhat.com
-  # @case_id OCP-34223
-  @upgrade-check
-  @qeci
-  @admin
-  @4.16 @4.15 @4.14 @4.13 @4.12 @4.11 @4.10 @4.9
-  @vsphere-ipi @openstack-ipi @nutanix-ipi @ibmcloud-ipi @gcp-ipi @baremetal-ipi @azure-ipi @aws-ipi @alicloud-ipi
-  @vsphere-upi @openstack-upi @nutanix-upi @ibmcloud-upi @gcp-upi @baremetal-upi @azure-upi @aws-upi @alicloud-upi
-  @singlenode
-  @upgrade
-  @network-ovnkubernetes @network-openshiftsdn
-  @proxy @noproxy @disconnected @connected
-  @s390x @ppc64le @heterogeneous @arm64 @amd64
-  @rosa @aro @osd_ccs
-  @hypershift-hosted
-  Scenario: kube-apiserver and openshift-apiserver should have zero-disruption upgrade
```

Comment:
The `Scenario: kube-apiserver and openshift-apiserver should have zero-disruption upgrade - prepare` should be removed in pair.

Reply:
@liangxia We have only this case 
```
grep -inr "Scenario: kube-apiserver and openshift-apiserver should have zero-disruption upgrade" .
./features/upgrade/api_auth/upgrade.feature:348:  Scenario: kube-apiserver and openshift-apiserver should have zero-disruption upgrade - prepare
rahulgangwar@rgangwar-mac verification-tests % 

```

Reply:
One scenario is `xxxxxx - prepare`, used to prepare upgrade data.
And another one is `xxxxxx`, used to verify the data after upgrade.
Generally they are used in pair. So if the scenario is no longer used/valid, we should remove them in pair, too.

Reply:
@liangxia For this scenario we do verify from this case, in this verify the logs to see there is no disruption and we prepare logs in upgrade pipeline, earlier we use shell script which continuously run with upgrade and update the logs in configmap. When the upgrade pipeline changes, we made changes in this also we  continuously run prepare logs and put in configmap https://issues.redhat.com/browse/OCPQE-8556. And verify from PstCheck case. We don't have case in pair

Reply:
The case OCP-34223 has been consolidated in function [waitClusterReady](https://gitlab.cee.redhat.com/aosqe/jenkins-jcasc-n/-/merge_requests/820/diffs#dae78ff7eca99d824f75ab4365c579293e1b3e05_474_522) of upgrade pipeline, it is executed in UpgradeRunner utils.waitClusterReady(env.VERSION, checkAPIServer:true, maxWaitTime:120) https://gitlab.cee.redhat.com/aosqe/jenkins-jcasc-n/-/blame/master/jobs/ocp-upgrade/upgrade-runner.jenkinsfile#L239

Reply:
> So there is no prepare scenario for this job in cucushift.

Check a few lines above in the same file, you will know what I'm talking about.

Reply:
@liangxia We know what you are saying, we don't have this case in pair.

Reply:
```
 grep -inr "Scenario: kube-apiserver and openshift-apiserver should have zero-disruption upgrade" .
./features/upgrade/api_auth/upgrade.feature:348:  Scenario: kube-apiserver and openshift-apiserver should have zero-disruption upgrade - prepare
rahulgangwar@rgangwar-mac verification-tests % grep -inr "zero-disrupt" .           
./features/upgrade/api_auth/upgrade.feature:348:  Scenario: kube-apiserver and openshift-apiserver should have zero-disruption upgrade - prepare
rahulgangwar@rgangwar-mac verification-tests % grep -inr "zero-disrupt" ../cucushift 
rahulgangwar@rgangwar-mac verification-tests % 
```

Reply:
@liangxia Sorry we missed that, updated